src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Models/BankIdLoginApiCancelRequest.cs

```diff
@@ -10,8 +10,8 @@ internal BankIdLoginApiCancelRequest()
         }
 
         [Required]
-        public string OrderRef { get; set; }
+        public string? OrderRef { get; set; }
```

Comment:
Why can this be null? First it is marked as required, second - what is the context to use cancel without the orderref? :)

Reply:
We do require it, but as this is something passed from the client, it can be null. If the client don't set a value for it, it will be null. The required flag only ells validation it is required. I know it looks strange, but the nullable stuff is more "can this ever be in a state where it is set to null", and it can be in this case :)